Documentation/platforms/xtensa/esp32s3/boards/esp32s3-devkit/index.rst

```diff
@@ -0,0 +1,58 @@
+==============
+ESP32S3-DevKit
+==============
+
+The `ESP32S3 DevKit <https://docs.espressif.com/projects/esp-idf/en/latest/esp32s3/hw-reference/esp32s3/user-guide-devkitc-1.html>`_ is a development board for the ESP32-S3 SoC from Espressif, based on a ESP32-S3-WROOM-1 module.
+
+.. list-table::
+   :align: center
+
+   * - .. figure:: esp32-s3-devkitc-1.png
+          :align: center
+
+Features
+========
+
+  - ESP32-S3-WROOM-1 Module
+  - USB-to-UART bridge via micro USB port
+  - Power LED
+  - EN and BOOT buttons (BOOT accessible to user)
+  - SPI FLASH (size varies according to model
+
+Serial Console
+==============
+
+UART0 is, by default, the serial console.  It connects to the on-board
+CP2102 converter and is available on the USB connector USB CON8 (J1).
+
+It will show up as /dev/ttyUSB[n] where [n] will probably be 0 (is it 1
+on my PC because I have a another device at ttyUSB0).
```

Comment:
Documentation should not be written in first person.
Consider rewriting this sentence about personal use case or simply remove it.